.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule ".husky"]
+	path = .husky
+	url = git@github.com:shardeum/git-hooks
```

I was running into issue here because my ssh key wasn't configured correctly, but since this is a public repo switching this to https should work for everyone without them needing extra config. 
```suggestion
	url = https://github.com/shardeum/git-hooks.git
```